web/source/dom/domEventHandlers.ts

```diff
@@ -32,7 +32,7 @@ namespace com.keyman.dom {
 
       this.focusTimer = window.setTimeout(function() {
         this.focusing=false;
-      }.bind(this), 1000)
+      }.bind(this), 50)
```

Comment:
Would it be better to define a constant?

Reply:
In theory yes but not sure that in practice it makes much difference here 😁